src/people/jess-mitchell.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Jess Mitchell
 job: Senior Manager, Research + Design
-pronouns: i opt out of choosing -- ask me why!
+pronouns: I opt out of choosing pronouns — ask me why?
 email: jmitchell@ocadu.ca
-website: https://jessmitchell.com
+website: https://jesshmitchell.com
```

Comment:
I had fixed these in main not that long ago. Did they get lost in an intermediate change somewhere? It would be good to understand why that might have happened, so that we can avoid similar issues in the future.

Reply:
This is a merge of your changes (made in dev) into main.

Reply:
That's interesting, because the change is already in main. 

https://github.com/inclusive-design/idrc/commit/aea8f7b1ec485bbdfdcedb2c76f9c1411c8adb9e

Reply:
Has main been merged into this branch?

Reply:
E.g. is this up to date with changes from main?

Reply:
That might explain it - dev might not have been up to date with main. Also it could be this change could have been done twice - work done directly to Main, and another to Dev and never merged.